src/cloudevent/partials/alerts/app-distribution-on-new-tester-ios-device-published.ts

```diff
@@ -0,0 +1,35 @@
+import {DeepPartial, MockCloudEventPartials} from '../../types';
+import {FirebaseAlertData} from 'firebase-functions/lib/v2/providers/alerts';
+import {AppDistributionEvent, NewTesterDevicePayload} from 'firebase-functions/lib/v2/providers/alerts/appDistribution';
+import {CloudFunction} from 'firebase-functions/lib/v2';
+import {getEventFilters, getEventType, PROJECT_ID} from '../helpers';
+
+export const alertsAppDistributionOnNewTesterIosDevicePublished:
+  MockCloudEventPartials<FirebaseAlertData<NewTesterDevicePayload>, FirebaseAlertData<NewTesterDevicePayload>> = {
+  generatePartial(
+    cloudFunction: CloudFunction<FirebaseAlertData<NewTesterDevicePayload>>):
+    DeepPartial<AppDistributionEvent<NewTesterDevicePayload>> {
+    const source = `//firebasealerts.googleapis.com/projects/${PROJECT_ID}`;
+    const now = new Date().toISOString();
+
+    return {
+      source,
+      type: getEventType(cloudFunction),
+      data: {
+        createTime: now,
+        endTime: now,
+        payload: {
+          ['@type']: 'com.google.firebase.firebasealerts.NewTesterDevicePayload',
```

Comment:
By the way, these are also made up values.

Reply:
Actually thats a good point, I tossed something random for `device model name` and `device identifier`, but I need to update this with something more realistic!!!